packages/sdks/snippets/react/src/components/customColumnsInfo.ts

```diff
@@ -0,0 +1,26 @@
+import { RegisteredComponent } from '@builder.io/sdk-react';
+import CustomColumns from './CustomColumns';
+
+export const customColumnsInfo: RegisteredComponent = {
+  name: 'MyColumns',
+  component: CustomColumns,
+  shouldReceiveBuilderProps: {
+    builderBlock: true,
+  },
+  inputs: [
+    {
+      name: 'columns',
+      type: 'array',
+      broadcast: true,
+      hideFromUI: true,
+      defaultValue: [
+        {
+          blocks: [],
+        },
+        {
+          blocks: [],
+        },
+      ],
+    },
+  ],
+};
```

Comment:
Can we merge each one of these component info files into component file itself? so move this to `CustomColumns.tsx`, etc.

Reply:
done